src/main/msp/msp.c

```diff
@@ -173,6 +175,11 @@ typedef enum {
 
 #define RTC_NOT_SUPPORTED 0xff
 
+typedef enum {
+    DEFAULTS_TYPE_BASE = 0,
+    DEFAULTS_TYPE_CUSTOM,
+} defaultsType_e;
```

Comment:
not sure about the use of `TYPE` here.  seems like a noise-word, like `data` or `info`.  perhaps `defaultsMode` ?

Reply:
Can you explain a bit more how this would be a 'mode', or how you think that 'info' isn't something that should be added to every data type containing information

Reply:
'data', 'type', 'details' and 'info' are often suffixes that are added to structures or variables when the name isn't thoroughly thought out, this often happens when you just need to get code working.  Ideally authors using those suffixes should likely go back and rename them after thinking a little bit harder about their name and purpose.   there's no hard-and-fast rule, but quite a common occurrence. 

I'm not sure that `mode` is much better than `type`, but when the code processed the data it does it one of two ways (`base` or `custom`) which in my head indicates it has a "mode" of operation, likely there's a better name out there.   don't get too hung up on it though if you can't thing of one and don't prefer `mode`.